### DISCUSSION TITLE
1.5.0rc1

### DISCUSSION THREAD

**Discussion by villebro** [apache/superset#19512] [General]
Hello Superset Community,

This is a call for the vote to release Apache Superset version 1.5.0.

The release candidate:
https://dist.apache.org/repos/dist/dev/superset/1.5.0rc1/

Git tag for the release:
https://github.com/apache/superset/tree/1.5.0rc1

The Change Log for the release:
https://github.com/apache/superset/blob/1.5.0rc1/CHANGELOG.md

The Updating instructions for the release:
https://github.com/apache/superset/blob/1.5.0rc1/UPDATING.md

public keys are available at:

https://www.apache.org/dist/superset/KEYS

The vote will be open for at least 72 hours or until the necessary number
of votes are reached.

Please vote accordingly:

[ ] +1 approve
[ ] +0 no opinion
[ ] -1 disapprove with the reason

Thanks,
The Apache Superset Team
--------------------------

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/1.5.0rc1'>1.5.0rc1</a>.</em>

**Comment by jeverling** [comment]
Upgrading from 1.4.2 (Docker setup using PostgreSQL) fails, see https://github.com/apache/superset/issues/19100

With a fresh DB it works fine, although entering/editing text into the Advanced JS modals (e.g. tooltip) is still broken (but pasting works as a workaround).

**Comment by villebro** [comment]
Thanks for raising this @jeverling , I'll look into this asap.

**Comment by kamalkeshavani-aiinside** [comment]
Is there a way we can disable this warning?
`filter_box will be deprecated in a future version of Superset. Please replace filter_box by dashboard filter components.`
With [this](https://github.com/apache/superset/issues/18959) issue still open, we can't move to native filters yet.

**Comment by villebro** [comment]
Thanks @kamalkeshavani-aiinside - I added the 1.5 label to the issue, let's see if we can tackle that and put forth 1.5.0rc2 with a fix.

**Comment by villebro** [comment]
@jeverling fix PR for the migration issue: #19543

**Comment by villebro** [comment]
Please check out the discussion for 1.5.0rc2: #19623
